services/supabase/migrations/20260626000000_support_individual_creators_and_aid_category.sql

```
@@ -0,0 +1,75 @@
+COMMENT ON COLUMN public.profiles.headline IS
+  'Short public tagline shown on individual creator pages';
+COMMENT ON COLUMN public.profiles.country IS
+  'ISO 3166-1 alpha-3 country code for the creator''s primary location';
+COMMENT ON COLUMN public.profiles.website_url IS
+  'Optional public website for individual creators';
+COMMENT ON COLUMN public.profiles.social_links IS
+  'Public social links for individual creators (e.g. twitter, instagram)';
+
+CREATE INDEX IF NOT EXISTS idx_profiles_creator_entity_type
+  ON public.profiles (creator_entity_type)
+  WHERE creator_entity_type IS NOT NULL;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)